types/webpack-sources/index.d.ts

```diff
@@ -1,15 +1,14 @@
-// Type definitions for webpack-sources v0.1.2
+// Type definitions for webpack-sources 0.1
```

Comment:
I haven't kept up with DefinitelyTyped's update, Is this mean we don't need `v` prefix?